modules/claims_api/config/routes.rb

```diff
@@ -59,6 +59,12 @@
     end
   end
 
+  namespace :v2, defaults: { format: 'json' } do
+    mount OkComputer::Engine, at: '/healthcheck'
+
+    get '/veteran-identifier', to: 'veteran_identifier#show'
```

Comment:
possible a nitpick but i wonder if this should be `veteran-id` instead of `veteran-identifier`

it really depends on what we want to put in other endpoints like
```
/v2/veterans/{veteranIdentifier}/claims
```
or 
```
/v2/veterans/{veteranId}/claims
```

the short "id" is just something im more used to i guess, but either way as long as theyre consistent is my larger opinion.

Reply:
I'm fine with `veteran-id`.  I'll switch it.